src/index.js

```diff
@@ -19,17 +20,29 @@ function onDomLoaded() {
   ROOT_ELEMENT = document.createElement('div');
   ROOT_ELEMENT.id = 'gfsdk_root_new';
   window.document.body.appendChild(ROOT_ELEMENT);
-  ReactDOM.render(
-    <Provider store={store}>
-      <App label={getLabel()} />
-    </Provider>,
-    ROOT_ELEMENT,
-  );
+  if (process.env.LOCAL_DEV === true || !window.docomo.SENTRY_ENABLE) {
+    window.docomo.SENTRY_URL = null;
+  }
+
+  Raven.config(window.docomo.SENTRY_URL, {
+    captureUnhandledRejections: true,
```

Comment:
We can add this to parameters (not tags):
`release: version.build,`

after the import of version
`import version from './version';`